### ISSUE TITLE
phony.rb uses #blank?, but library doesn't require Active Support core extensions

### ISSUE THREAD

**Issue by rlue** [floere/phony#403]
17ee392 introduced the following class:

```ruby
class SplittingError < ArgumentError
  def initialize number
    super %Q{Phony could not split the given number. Is #{number.blank? ? 'it' : number.inspect} a phone number?}
  end
end
```

The use of `Object#blank?` on the third line raises a `NoMethodError` unless `active_support/core_ext/object` is required.

Not sure whether @floere would want to rewrite this method or add the dependency in the gemspec, so I'm reporting it here instead of submitting a PR.

**Comment by floere** [comment]
@rlue Oh my – thanks so much for noticing! It's in master and I'll release a fix within the hour 😊

**Comment by floere** [comment]
Thanks again – released in 2.16.4 🚀
